net/model/CommentBox.h

```diff
@@ -0,0 +1,56 @@
+//
```

Comment:
I've created a folder with name: UI custom obj. Name is so-so. For now there are such classes (files .h and .m):
CommentBox
ChangerBox
UIColor+Bawl
AvatarMaskView
AvatarView